plugins/drill-coverage-plugin/src/adminPartMain/kotlin/CoverageController.kt

```diff
@@ -32,6 +32,10 @@ class CoverageController(sender: WsService, agentInfo: AgentInfo, id: String) :
             ActionType.DROP_SCOPE -> {
                 checkoutScope("", agentState)
             }
+            ActionType.TOGGLE_SCOPE -> {
+                val scope = getScopeOrNull("${agentInfo.id}-${agentInfo.buildVersion}-${action.payload.scopeName}")
```

Comment:
I think u should extract this pattern to common file/utils? and use it like buildKey(1,2,3)